### ISSUE TITLE
Upgrading devices to iOS 17 beta

### ISSUE THREAD

**Issue by vashworth** [flutter/flutter#132251]
iOS 17 physical iOS devices have an entirely new device connectively stack and therefore we had to build [new workflows](https://github.com/flutter/flutter/pull/131865) for it.

We would like to be able to test iOS 17 beta in the devicelab before the official release (probably sometime September).

First, [Xcode 15 needs to be uploaded to CI](https://github.com/flutter/flutter/issues/132128) - I plan to do this this week or next - currently blocked by not being able to download iOS 17 beta 5 simulator runtime.

I'm not sure how many devices we should upgrade, but at least one.

Urgent because new workflows cannot be completely tested without a physical iOS 17 device and new workflow is most likely going to be cherrypicked into beta this week and hit stable next week.

**Comment by vashworth** [comment]
cc @yusuf-goog

**Comment by vashworth** [comment]
I also made an issue for upgrading to official iOS 17 once it's released: https://github.com/flutter/flutter/issues/132252

**Comment by yusuf-goog** [comment]
Do you want a sample set updated to the beta, or are requesting the whole fleet update to beta and then to final release?

**Comment by ricardoamador** [comment]
@yusuf-goog @kentnguyen99 feel free to reprioritize this as needed.

**Comment by vashworth** [comment]
We're not going to be using beta in CI, so closing in favor of https://github.com/flutter/flutter/issues/132252
